DataBiosphere/leonardo
# [IA-4401] Bump terra-azure-relay-listeners to fix runtime autopause

See https://github.com/DataBiosphere/terra-azure-relay-listeners/pull/42

This version of terra-azure-relay-listeners checks requests for the custom HTTP header `X-SetDateAccessedInspector-Action=ignore`, and does not send a PATCH updateDateAccessed if the header is present. This prevents Leo's MetricsMonitor calls (through JupyterDAO and WelderDAO) from blocking Azure runtime autopause.

__Jira ticket__: https://broadworkbench.atlassian.net/browse/IA-4401

### Testing

- Create an Azure runtime with an autopause value 30 minutes or higher
- Open JupyterLab and issue a command
- Wait 30m (or the autopause duration)
- Runtime is paused

See test cases listed here: https://github.com/DataBiosphere/terra-azure-relay-listeners/pull/42

### Who tested and where

  - _NB: Rerun automation tests on this PR by commenting `jenkins retest` or `jenkins multi-test`._
